### DISCUSSION TITLE
Dark background for a label

### DISCUSSION THREAD

**Discussion by zanac** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#688] [General]
Hi,

how can i make a "white rectangular" for fill a label, and of course use a black font inside to see the room name much more visible? In the screenshot the name "Corridoio" don't appare...

![image](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/21194919/54d72400-2c68-4e69-a917-f74c9adc0b6d)

**Comment by PiotrMachowski** [comment]
It should be possible to do using card mod

**Reply by zanac** [reply]
Ok have you Got a sample? Which CSS Class i should overwrite?
